src/libcommon/log/sentry/sentryhandler.cpp

```diff
@@ -172,28 +189,39 @@ std::shared_ptr<SentryHandler> SentryHandler::instance() {
     return _instance;
 }
 
-void SentryHandler::init(SentryProject project, int breadCrumbsSize) {
+void SentryHandler::init(KDC::AppType appType, int breadCrumbsSize) {
     if (_instance) {
         assert(false && "SentryHandler already initialized");
         return;
     }
 
     _instance = std::shared_ptr<SentryHandler>(new SentryHandler());
 
-    if (project == SentryProject::Deactivated) {
+    if (appType == KDC::AppType::None) {
         _instance->_isSentryActivated = false;
         return;
     }
 
+    _appType = appType;
+
+    bool isSet = false;
+    if (CommonUtility::envVarValue("KDRIVE_DEBUG_SENTRY_CRASH_CB", isSet); isSet) {
+        _debugCrashCallback = true;
+    }
+    isSet = false;
+    if (CommonUtility::envVarValue("KDRIVE_DEBUG_SENTRY_BEFORE_SEND_CB", isSet); isSet) {
```

Comment:
Maybe a comment about the fact that, if set, the second value will override the first one.

Reply:
Done